CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Implement sleep with some wakeup methods for `esp32` (#574)
 - Add a new RMT driver (#653, #667)
 - Add MCPWM DeadTime configuration (#406)
+- Added module-level documentation for all peripherals
```

Comment:
```suggestion
- Added module-level documentation for all peripherals (#680)
```

Reply:
Though I'm not sure if docs need a changelog 🤔

Reply:
I'm pretty sure that record should be there since:
1) Everything should be logged 😁 
2) Module-level documentation is a high priority job and milestone for us, so I think it deserves it's place in the `CHANGELOG`